alyssa_mckee/Django/Login_Registration/apps/login_registration/templates/login_registration/index.html

```diff
@@ -0,0 +1,79 @@
+<!doctype html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Log & Reg Home</title>
+	<style>
+		label{
+			display: inline-block;
+			width: 100px;
+			margin-bottom: 10px;
+		}
+		.error{
+			color: red;
+		}
+	</style>
+</head>
+<body>
+	<form action="{% url 'login' %}" method="POST">
+	{% csrf_token %}
+		<h3>Login: </h3>
+		
+			{% if messages %}
+				<ul>{% for message in messages %}
+						{% if message.extra_tags == "login" %}
+							<li class="error">{{message}}</li>
+						{%endif%}
+					{%endfor%}
+				</ul>
+			{%endif%}
+
+		<div>
+			<label for="email">Email:</label>
+			<input type="text" name="email" id="email">
+		</div>
+		<div>
+			<label for="password">Password:</label>
+			<input type="password" name="password" id="password">
+		</div>
+		<button>Login</button>
+	
+	</form>
+	<hr />
+	<form action="{% url 'register' %}" method="POST">
+		{% csrf_token %}
+		<h3>Register:</h3>
+			{% if messages %}
+				<ul>{% for message in messages %}
+						{% if message.extra_tags == "registration" %}
```

Comment:
Great job utilization the messaging frameworks tag feature to show errors onto specific forms!